catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/source/sierra/Language.scala

```diff
@@ -0,0 +1,3 @@
+package uk.ac.wellcome.transformer.source.sierra
+
+case class Country(code: String, name: String)
```

Comment:
and this in a file called Language.scala

Reply:
Bwaaaaaah. Fixing, thanks.